_posts/2018-07-30-ARIA-properties.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: ARIA Properties
```

Comment:
Since this article will become a snapshot in time (references ARIA 1.1 spec, but work is underway for ARIA 1.2) should the version number be referenced somehow, or should there be a link (in the intro or outro) that goes to the latest working draft?

this comment applies to the second post as well.

Reply:
Left "Properties" as the title, but added "1.1" in the first sentence and in the table of specs, as well as added a link to the latest working draft in the intro.